src/docs/events.yaml

```diff
@@ -0,0 +1,204 @@
+paths:
+  /events/{id}:
+    get:
+      tags: 
+      - "Events"
+      summary: "Get event by id"
+      description: "Returns a single event"
+      operationId: "getEventById"
+      produces:
+      - "application/json"
+      parameters:
+        - name: "id"
+          in: "path"
+          description: "ID of the event to return"
+          required: true
+          type: "integer"
```

Comment:
The id is actually a string, not an integer. Change this wherever you use an Id.